app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  validates :text, presence: true
+  belongs_to :post
+  belongs_to :user
```

Comment:
- [ ] As I already told you that the author is the alias of the user then it would be better if you make the belongs_to relationship of the comment with the author 👍 After making the changes it would look like the below ⬇️

```ruby
belongs_to :post
belongs_to :author, class_name: 'User'
```